keyboards/cannonkeys/satisfaction75/keymaps/boy_314/satisfaction75.c

```diff
@@ -0,0 +1,473 @@
+#include "satisfaction75.h"
```

Comment:
I dont imagine we would accept this PR as-is, due to the duplication of this file. From discord, i understand its due to recent split encoder changes, and #7568 not being merged. But as this is the main handling for via, and is due for removal in #7268, duplicating the file makes things more complicated long term.

Reply:
This file should have been removed. Was trying to mess around with the default encoder function when plugging in the keyboard, but I couldn't get it to work.